cmd/server/main.go

```diff
@@ -592,6 +591,7 @@ func (p *Proxy) runHealthServer(o *ProxyRunOptions, server *server.ProxyServer)
 	})
 
 	muxHandler := http.NewServeMux()
+	muxHandler.Handle("/metrics", promhttp.Handler())
 	muxHandler.HandleFunc("/healthz", livenessHandler)
 	muxHandler.HandleFunc("/ready", readinessHandler)
```

Comment:
shouldn't this be `/readyz`?

Reply:
shrug, Kubelet doesn't care. Happy to have it changed if we believe that is more standard.